lexicon.md

```diff
@@ -0,0 +1,130 @@
+|**Interoperability**||The ability of different systems and organizations to work together (inter-operate) by using standardized data formats and protocols.|
+
+## L
+
+|Term|Abbriviation|Description|
+|---|---|---|
+|**Linked Data Platform**|LDP|A set of W3C standards for building interoperable web applications using Linked Data principles.|
+|**Linked Data**||A method of publishing structured data so that it can be interlinked and become more useful through semantic queries.|
+
+## M
+
+|Term|Abbriviation|Description|
+|---|---|---|
+|**Matcher**||A component or process that selects resources or data based on specified criteria.|
+
+## O
+
+|Term|Abbriviation|Description|
+|---|---|---|
+|**Object**||In RDF, the object is the third part of a triple that describes the value or entity related to the subject via the predicate. *(For Example: In the triple `<John, hasFriend, Jane>`, "Jane" is the object.)*|
+|**Origin**||The source domain or URL from which a request is made.|
+|**Owner**||The individual or entity that has control over a resource or Solid pod.|
+
+## P
+
+|Term|Abbriviation|Description|
+|---|---|---|
+|**Permission**||The authorization given to a user or application to access and manipulate resources within a Solid pod.|
+|**Policy**|| A set of rules or permissions that govern access to resources.
+|**Policy Type/User Type**||Categories or roles that determine the policies applicable to different users or groups.|
+|**Profile**||A resource that describes a Solid user, typically containing information about the user and links to other resources.|
+|**Properties**||Characteristics or attributes of a resource.|
+|**Prohibation**||A restriction or denial of certain actions on a resource.|
+
+## R
+
+|Term|Abbriviation|Description|
+|---|---|---|
+|**Resource Description Framework**|RDF|A standard model for data interchange on the web, which Solid uses to structure and link data.|
+|**Read**||The permission to view a resource.|
+|**Resource**||Any item (file, document, data) stored in a Solid pod, which can be accessed or manipulated through Solid protocols.|
+|**Rule**||A statement that defines conditions and constraints for actions and permissions.|
+
+## S
+
+|Term|Abbriviation|Description|
+|---|---|---|
+|**Social Linked Data**|SOLID|A web decentralization project, aiming to give users control over their personal data by storing it in decentralized pods.|
+|**Solid App**||An application designed to interact with Solid pods and utilize Solid protocols for data access and management.|
+|**Solid Client**||An application or user agent that interacts with a Solid server to read and write data.|
+|**Solid OIDC**|Solid OpenId Connect|An authentication layer on top of the OAuth 2.0 protocol, used for secure, federated login in the Solid ecosystem.|
```

Comment:
```suggestion
|**Solid OpenId Connect**|Solid OIDC|An authentication layer on top of the OAuth 2.0 protocol, used for secure, federated login in the Solid ecosystem.|
```
abbreviation should not be longer than the original name?

Reply:
Yeah, no, you're right, is probably a mistake on my part ^^'